### ISSUE TITLE
Incorrect currency displayed on stats on first load

### ISSUE THREAD

**Issue by jostnes** [woocommerce/woocommerce-ios#6301]
**Describe the bug**
Using a store that isn't using $ as currency, when loading the stats screen for the first time, the currency value is $ instead of the store currency.

**To Reproduce**
1. Before moving to the next steps, kill the app
2. Launch Woo App
3. Notice on the stats page, the currency is $ (my store is set to use RM)
4. Tap to view other tabs (e.g `This Week`) and return to `Today`
5. Currency is updated and is now correct

**Screenshots**

https://user-images.githubusercontent.com/17252150/155471429-cf24333e-4130-47d9-9e87-49ac2ad8c93c.MP4

**Expected behavior**
The currency should be displayed correctly on the first load

**Isolating the problem (mark completed items with an [x]):**
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Mobile Environment**
Please include:
- Device: iPhone 13 mini
- iOS version: 15.0
- WooCommerce iOS version: Reproducible on both 8.6 (current beta) and 8.5 (current production)

**WordPress Environment**
<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin.
```
</details>

**Comment by jaclync** [comment]
Just a note that I wasn't able to reproduce this issue when relaunching the app with a non-USD store, but I can reproduce this issue by switching to a non-USD store

**Comment by erricgunawan** [comment]
We received a report from 4848039-zen but are unsure if it's the same issue as listed here.

The reason is, we asked them to tap the "This Month" tab and then go back to the "Today" tab, and check if they then see the correct currency.

Their response is:

> Switching between the day month and year tabs does not resolve the issue. The correct current is displayed on the orders list and order details screens.

We also notice there's an ongoing `WP.com Cache has discrepancies` and `Cron Size: 52 The cron array size is pretty large;` on the site's Jetpack Debugger. 

Should we ask the user to still wait until this fix is released?

Or is there something else that might happen here? Plugin conflict? (I see they have `All In One WP Security`). 



<details>
  <summary>I included their System Status Report here:</summary>

```
### WordPress Environment ###

WordPress address (URL): (redacted)
Site address (URL): (redacted)
WC Version: 6.3.1
REST API Version: ✔ 6.3.1
WC Blocks Version: ✔ 6.9.0
Action Scheduler Version: ✔ 3.4.0
WC Admin Version: ✔ 3.2.1
Log Directory Writable: ✔
WP Version: 5.9.2
WP Multisite: –
WP Memory Limit: 8 GB
WP Debug Mode: –
WP Cron: ✔
Language: en_GB
External object cache: –

### Server Environment ###

Server Info: Apache
PHP Version: 7.4.28
PHP Post Max Size: 64 MB
PHP Time Limit: 90
PHP Max Input Vars: 1000
cURL Version: 7.68.0
OpenSSL/1.1.1f

SUHOSIN Installed: –
MySQL Version: 5.5.5-10.3.34-MariaDB-0ubuntu0.20.04.1
Max Upload Size: 32 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 6.3.1
WC Database Prefix: taglm_
Total Database Size: 227.04MB
Database Data Size: 140.93MB
Database Index Size: 86.11MB
taglm_woocommerce_sessions: Data: 6.02MB + Index: 0.14MB + Engine InnoDB
taglm_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
taglm_woocommerce_order_items: Data: 1.52MB + Index: 0.23MB + Engine InnoDB
taglm_woocommerce_order_itemmeta: Data: 7.52MB + Index: 6.03MB + Engine InnoDB
taglm_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
taglm_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_woocommerce_payment_tokens: Data: 0.05MB + Index: 0.02MB + Engine InnoDB
taglm_woocommerce_payment_tokenmeta: Data: 0.08MB + Index: 0.09MB + Engine InnoDB
taglm_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_actionscheduler_actions: Data: 19.05MB + Index: 22.25MB + Engine InnoDB
taglm_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_actionscheduler_logs: Data: 12.02MB + Index: 10.03MB + Engine InnoDB
taglm_aiowps_debug_log: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_aiowps_events: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_aiowps_failed_logins: Data: 1.52MB + Index: 0.00MB + Engine InnoDB
taglm_aiowps_global_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_aiowps_login_activity: Data: 0.14MB + Index: 0.00MB + Engine InnoDB
taglm_aiowps_login_lockdown: Data: 6.52MB + Index: 0.00MB + Engine InnoDB
taglm_aiowps_permanent_block: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_atum_order_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_atum_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_atum_product_data: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
taglm_automatewoo_abandoned_carts: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
taglm_automatewoo_customers: Data: 0.31MB + Index: 0.52MB + Engine InnoDB
taglm_automatewoo_customer_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_automatewoo_events: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
taglm_automatewoo_guests: Data: 0.11MB + Index: 0.19MB + Engine InnoDB
taglm_automatewoo_guest_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_automatewoo_logs: Data: 0.14MB + Index: 0.20MB + Engine InnoDB
taglm_automatewoo_log_meta: Data: 1.52MB + Index: 0.47MB + Engine InnoDB
taglm_automatewoo_queue: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
taglm_automatewoo_queue_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_commentmeta: Data: 0.06MB + Index: 0.03MB + Engine InnoDB
taglm_comments: Data: 5.52MB + Index: 2.72MB + Engine InnoDB
taglm_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_mailchimp_carts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_mailchimp_jobs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_options: Data: 19.30MB + Index: 2.13MB + Engine InnoDB
taglm_postmeta: Data: 23.55MB + Index: 20.78MB + Engine InnoDB
taglm_posts: Data: 9.47MB + Index: 2.64MB + Engine InnoDB
taglm_sib_model_contact: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_sib_model_country: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_social_users: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_stock_log: Data: 0.48MB + Index: 0.00MB + Engine InnoDB
taglm_termmeta: Data: 0.08MB + Index: 0.11MB + Engine InnoDB
taglm_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_term_relationships: Data: 0.17MB + Index: 0.14MB + Engine InnoDB
taglm_term_taxonomy: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
taglm_tm_taskmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_tm_tasks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_usermeta: Data: 3.50MB + Index: 3.94MB + Engine InnoDB
taglm_users: Data: 0.16MB + Index: 0.16MB + Engine InnoDB
taglm_wcs_payment_retries: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_wc_admin_notes: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_wc_customer_lookup: Data: 0.25MB + Index: 0.25MB + Engine InnoDB
taglm_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_wc_gpf_render_cache: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_wc_order_product_lookup: Data: 1.52MB + Index: 0.50MB + Engine InnoDB
taglm_wc_order_stats: Data: 0.30MB + Index: 0.23MB + Engine InnoDB
taglm_wc_order_tax_lookup: Data: 0.19MB + Index: 0.17MB + Engine InnoDB
taglm_wc_product_attributes_lookup: Data: 0.19MB + Index: 0.22MB + Engine InnoDB
taglm_wc_product_meta_lookup: Data: 1.52MB + Index: 1.17MB + Engine InnoDB
taglm_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_woocommerce_gc_activity: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
taglm_woocommerce_gc_cards: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
taglm_woocommerce_gc_cardsmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_woocommerce_gpf_google_taxonomy: Data: 1.52MB + Index: 0.13MB + Engine InnoDB
taglm_wpmailsmtp_emails_log: Data: 1.52MB + Index: 0.03MB + Engine InnoDB
taglm_wpmailsmtp_tasks_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_wt_iew_action_history: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_wt_iew_cron: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_wt_iew_ftp: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_wt_iew_mapping_template: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_xtraderstockmanager_plugin_category_sync: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_xtraderstockmanager_plugin_product_sync: Data: 0.11MB + Index: 0.00MB + Engine InnoDB
taglm_xtraderstockmanager_plugin_run_data: Data: 0.48MB + Index: 0.00MB + Engine InnoDB
taglm_yith_wcwl: Data: 0.06MB + Index: 0.02MB + Engine InnoDB
taglm_yith_wcwl_lists: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_yoast_indexable: Data: 11.52MB + Index: 8.50MB + Engine InnoDB
taglm_yoast_indexable_hierarchy: Data: 1.42MB + Index: 0.67MB + Engine InnoDB
taglm_yoast_migrations: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_yoast_primary_term: Data: 0.06MB + Index: 0.03MB + Engine InnoDB
taglm_yoast_seo_links: Data: 0.31MB + Index: 0.17MB + Engine InnoDB

### Post Type Counts ###

attachment: 1520
aw_workflow: 8
blocks: 21
custom_css: 1
customize_changeset: 7
featured_item: 8
gift_card: 7
gtc_design: 5
gtc_gift_card: 2
gtc_transaction_log: 1
jp_img_sitemap: 2
jp_sitemap: 2
jp_sitemap_master: 2
nav_menu_item: 22
oembed_cache: 1
page: 25
post: 3
product: 252
product_variation: 1338
revision: 218
shop_bogof_rule: 1
shop_coupon: 9
shop_order: 2599
shop_order_refund: 63
shop_subscription: 19
wc_product_tab: 3
wp_global_styles: 1
wpcf7_contact_form: 2
ywtm_tab: 5

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (32) ###

PW Advanced Woo Reporting for "Brand Plugins" ADD-ON: by Proword – 1.1
PW Advanced Woo Reporting: by Proword – 6.0
Akismet Anti-Spam: by Automattic – 4.2.2
All In One WP Security: by Tips and Tricks HQ
Peter Petreski
Ruhul
Ivy – 4.4.10

AutomateWoo: by WooCommerce – 5.5.11
Contact Form 7: by Takayuki Miyoshi – 5.5.6
Custom Adobe Fonts (Typekit): by Brainstorm Force – 1.0.18
Jetpack: by Automattic – 10.7
reCaptcha for WooCommerce: by I Thirteen Web Solution – 2.29
Regenerate Thumbnails: by Alex Mills (Viper007Bond) – 3.1.5
SSH SFTP Updater Support: by TerraFrost
David Anderson + Team Updraft – 0.8.4

Bulk Table Editor for WooCommerce: by Consortia – 2.2.9
WooCommerce Admin: by WooCommerce – 3.2.1
WooCommerce Brands: by WooCommerce – 1.6.28
WooCommerce Buy One Get One Free: by Oscar Gare – 3.1.0
WooCommerce Custom Thank You Pages: by Jeroen Sormani – 1.0.5
WooCommerce Stripe Gateway: by WooCommerce – 6.3.0
WooCommerce Google Analytics Integration: by WooCommerce – 1.5.8
WooCommerce PayPal Payments: by WooCommerce – 1.7.0
WooCommerce Google Product Feed: by Ademti Software Ltd. – 10.5.3
WooCommerce Product Reviews Pro: by SkyVerge – 1.17.4
Sendinblue - WooCommerce Email Marketing: by Sendinblue – 2.0.34
WooCommerce Shipping & Tax: by WooCommerce – 1.25.23
Stock Manager for WooCommerce: by StoreApps – 2.8.3
WooCommerce Subscriptions: by WooCommerce – 4.0.2
WooCommerce Xero Integration: by WooCommerce – 1.7.41
WooCommerce: by Automattic – 6.3.1
Yoast SEO: by Team Yoast – 18.3
WP-Optimize - Clean, Compress, Cache: by David Anderson
Ruhani Rabin
Team Updraft – 3.2.2

YITH WooCommerce Gift Cards Premium: by YITH – 3.12.1
YITH WooCommerce Tab Manager Premium: by YITH – 1.9.0
YITH WooCommerce Wishlist: by YITH – 3.7.0

### Inactive Plugins (5) ###

Trustpilot-reviews: by Trustpilot – 2.5.901
Variation Swatches for WooCommerce: by Emran Ahmed – 1.1.19
Variation Swatches for WooCommerce - Pro: by Emran Ahmed – 1.1.18
WooCommerce Gift Cards: by WooCommerce – 1.10.0
XTrader StockManager: by XTrader – 3.43

### Dropin Plugins (1) ###

maintenance.php: maintenance.php

### Settings ###

API Enabled: –
Force SSL: –
Currency: GBP (£)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
gift_card (gift_card)
gift-card (gift-card)
grouped (grouped)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: ✔

### WC Pages ###

Shop base: [#63](63-zd-woothemes) - /shop/
Basket: [#62](62-zd-woothemes) - /cart/
Checkout: [#61](61-zd-woothemes) - /checkout/
My account: [#60](60-zd-woothemes) - /my-account/
Terms and conditions: [#845](845-zd-woothemes) - /terms-and-conditions/

### Theme ###

Name: Flatsome Child
Version: 3.0
Author URL:
Child Theme: ✔
Parent Theme Name: Flatsome
Parent Theme Version: 3.15.3
Parent Theme Author URL: http://www.uxthemes.com/
WooCommerce Support: ✔

### Templates ###

Overrides: flatsome/woocommerce/archive-product.php
flatsome/woocommerce/cart/cart-empty.php
flatsome/woocommerce/cart/cart-shipping.php
flatsome/woocommerce/cart/cart.php
flatsome/woocommerce/checkout/form-checkout.php
flatsome/woocommerce/checkout/form-coupon.php
flatsome/woocommerce/checkout/terms.php
flatsome/woocommerce/checkout/thankyou.php
flatsome/woocommerce/content-product-cat.php
flatsome/woocommerce/content-product.php
flatsome/woocommerce/content-single-product.php
flatsome/woocommerce/content-widget-product.php
flatsome/woocommerce/global/breadcrumb.php
flatsome/woocommerce/global/quantity-input.php
flatsome/woocommerce/global/wrapper-end.php
flatsome/woocommerce/global/wrapper-start.php
flatsome/woocommerce/loop/loop-end.php
flatsome/woocommerce/loop/loop-start.php
flatsome/woocommerce/loop/pagination.php
flatsome/woocommerce/loop/result-count.php
flatsome/woocommerce/loop/sale-flash.php
flatsome/woocommerce/myaccount/form-login.php
flatsome/woocommerce/myaccount/navigation.php
flatsome/woocommerce/notices/error.php
flatsome/woocommerce/notices/notice.php
flatsome/woocommerce/notices/success.php
flatsome/woocommerce/product-searchform.php
flatsome/woocommerce/single-product/photoswipe.php
flatsome/woocommerce/single-product/price.php
flatsome/woocommerce/single-product/product-image.php
flatsome/woocommerce/single-product/product-thumbnails.php
flatsome/woocommerce/single-product/rating.php
flatsome/woocommerce/single-product/related.php
flatsome/woocommerce/single-product/review.php
flatsome/woocommerce/single-product/sale-flash.php
flatsome/woocommerce/single-product/share.php
flatsome/woocommerce/single-product/short-description.php
flatsome/woocommerce/single-product/tabs/tabs.php
flatsome/woocommerce/single-product/title.php
flatsome/woocommerce/single-product/up-sells.php
flatsome/woocommerce/single-product-reviews.php
flatsome/woocommerce/single-product.php
flatsome/woocommerce/content-product_cat.php

### WooCommerce Google Product Feed feeds ###

google: Google merchant centre product feed
Type: google
Category filter: -
Categories: -

### WooCommerce Google Product Feed options ###

Include variations in feed: Enabled
Send "item group ID": -
Expanded schema markup: -
Debug key: f0cbe72f-b800-496e-b8a5-b5a0c292fcc1

### WooCommerce Google Product Feed fields ###

Title: Pre-populates from product product_title.
Product description: Main product description (full preferred) plus variation description
Availability (in stock products): Defaults to "in stock".
Availability (backordered products): Defaults to "in stock".
Availability (out of stock products): Defaults to "out of stock".
Condition: Defaults to "new".
Brand: Pre-populates from product_brand taxonomy.
Manufacturer Part Number (MPN): Pre-populates from product sku.
Product Type: Pre-populates from product_cat taxonomy.
Google Product Category:
Gender: Defaults to "male".
Age Group:
Colour: Pre-populates from pa_colour taxonomy.
Size: Pre-populates from pa_size taxonomy.
Size system: Defaults to "UK".
Product detail(s):

### WooCommerce Google Product Feed DB status ###

Database version: 15
Active database version: 15
wc_gpf_render_cache: Empty
woocommerce_gpf_google_taxonomy (en-GB): 5595 items

### Subscriptions ###

WCS_DEBUG: ✔ No
Subscriptions Mode: ✔ Live
Subscriptions Live URL: https://loadedmenswear.co.uk/
Subscription Statuses: wc-active: 12
wc-pending-cancel: 1
wc-cancelled: 6

WooCommerce Account Connected: ✔ Yes
Active Product Key: ❌ No
Custom Retry Rules: ✔ No
Custom Retry Rule Class: ✔ No
Custom Raw Retry Rule: ✔ No
Custom Retry Rule: ✔ No
Retries Migration Status: ✔ Completed
Report Cache Enabled: ✔ Yes
Cache Update Failures: ✔ 0 failure

### Store Setup ###

Country / State: United Kingdom (UK)

### Subscriptions by Payment Gateway ###

Stripe: wc-active: 12
wc-cancelled: 6
wc-pending-cancel: 1

### Payment Gateway Support ###

Stripe: products
refunds
tokenization
add_payment_method
subscriptions
subscription_cancellation
subscription_suspension
subscription_reactivation
subscription_amount_changes
subscription_date_changes
subscription_payment_method_change
subscription_payment_method_change_customer
subscription_payment_method_change_admin
multiple_subscriptions

PayPal: refunds

### Buy One Get One Free ###

Display eligible free gift(s) on: after_cart
Disable coupons: yes
Custom attributes: no
Rule [#34862](34862-zd-woothemes): title: "Half Price Sukerw"; type: "buy_a_get_b"; applies_to: " All Products "; action: "choose_from"; gift_products: " Product In list ( Sukrew Bloomsbury Brief ([#3453](3453-zd-woothemes)) | Sukrew Bristol Apex Brief ([#1046](1046-zd-woothemes)) | Sukrew Camden Brief ([#2266](2266-zd-woothemes)) | Sukrew Chelsea Brief ([#3478](3478-zd-woothemes)) | Sukrew Classic Brief ([#6107](6107-zd-woothemes)) | Sukrew Dorchester Apex Brief ([#2459](2459-zd-woothemes)) | Sukrew Monmouth Apex Brief ([#2836](2836-zd-woothemes)) | Sukrew Opal V Thong ([#5435](5435-zd-woothemes)) | Sukrew Ramsey Brief ([#1884](1884-zd-woothemes)) | Sukrew Rushden Brief ([#3951](3951-zd-woothemes)) | Sukrew Selby Brief ([#3963](3963-zd-woothemes)) | Sukrew Vauxhall Backless V Brief ([#1212](1212-zd-woothemes)) | Sukrew York Apex Brief ([#2468](2468-zd-woothemes)) ) "; min_quantity: "2"; free_quantity: "1"; discount: "50"; start_date: "2022-03-01 00:00:00"; end_date: "2022-03-31 23:59:59"; exclude_other_rules: "1"; exclude_coupon_validation: "1"

### WooCommerce PayPal Payments ###

Onboarded: ✔
Shop country code: GB
WooCommerce currency supported: ✔
PayPal card processing available in country: ✔
Pay Later messaging available in country: ✔
Webhook status: ✔
Vault enabled: ✔
Logging enabled: –
Reference Transactions: –
Used PayPal Checkout plugin: –

### Action Scheduler ###

Complete: 56,201
Oldest: 2022-02-08 17:39:10 +0000
Newest: 2022-03-11 16:40:56 +0000

Failed: 1
Oldest: 2021-05-08 08:52:52 +0100
Newest: 2021-05-08 08:52:52 +0100

Pending: 17
Oldest: 2022-03-11 18:00:00 +0000
Newest: 2022-04-09 13:25:40 +0100

### Status report information ###

Generated at: 2022-03-11 17:25:54 +00:00
```

</details>

**Comment by itsmeichigo** [comment]
Hi @erricgunawan - if the user cannot get the correct currency after switching between time ranges, it's likely that they have a different issue. But I think it's still worth it to ask them to try again with version 8.7 so we can know for sure if the fix in #6335 helps. Thanks!

**Comment by thabotswana** [comment]
The user replied saying they updated to version 8.7, but that didn't help.

![image1](https://user-images.githubusercontent.com/35628116/159690579-b4f5fcf5-fbe9-4419-8e18-d9cb913f6f54.png)

I scheduled another re-sync of their data to see if that helps.

**Comment by itsmeichigo** [comment]
There's one thing I missed from their device logs that there was an error synching their site settings: `Site settings sync had 3 error(s) for siteID 182095388: [Dotcom Error: [rest_setting_setting_group_invalid] Invalid setting group., Dotcom Error: [rest_setting_setting_group_invalid] Invalid setting group., Dotcom Error: [rest_setting_setting_group_invalid] Invalid setting group.]`. 

This is why we cannot get the correct currency on the app. The Orders tab shows the correct currency because we're displaying the orders' currency instead of using the one in the site setting.

I see that the team has tried a few ways to fix their Jetpack connection - I wonder why it hasn't worked out for them yet.

**Comment by erricgunawan** [comment]
Thanks for finding that error log @itsmeichigo !

I noticed JPDB keeps saying there are some discrepancies no matter how many times we tried to re-schedule a sync. 

This is the last one I checked:

```
WP.com Cache has discrepancies as of last checksum on 2022-03-11 17:01:53
```

Also, this is also in JPDB, although I'm unsure if it has anything to do with it:

```
Cron Size: 51 The cron array size is pretty large; this could indicate that cron is not working as expected. Big cron size isn't a sync problem by itself, it's just a warning that the site is poorly maintained/configured. In order to diagnose Sync issues we need to look at "Queue Sizes", whenever these are close to 0 it is an indication that Sync is healthy.
```

User also added this in their replies:

> I have noted that when I change store and then back again- it does display the direct currency but then when I close the app and open later it’s back to USD!

They also already tried to reinstall the app, but that doesn't help either.

Any idea for the next steps?

**Comment by itsmeichigo** [comment]
> I have noted that when I change store and then back again- it does display the direct currency but then when I close the app and open later it’s back to USD!

This is weird because it seems that we are using a common logic to re-fetch site settings when launching the app and switching stores. Is it possible to SSP to confirm this behavior?

**Comment by erricgunawan** [comment]
We have permission from the user to access their account and I can confirm this.

<details>
  <summary>Here's a screen recording I take:</summary>
  <br>
  
https://user-images.githubusercontent.com/1813857/160223566-8229e418-82ce-42b6-bc28-f908568f7958.mp4

</details>

<details>
 <summary>System Status Report</summary>
<br>
### WordPress Environment ###

WordPress address (URL): https://loadedmenswear.co.uk
Site address (URL): https://loadedmenswear.co.uk
WC Version: 6.3.1
REST API Version: ✔ 6.3.1
WC Blocks Version: ✔ 6.9.0
Action Scheduler Version: ✔ 3.4.0
WC Admin Version: ✔ 3.2.1
Log Directory Writable: ✔
WP Version: 5.9.2
WP Multisite: –
WP Memory Limit: 8 GB
WP Debug Mode: –
WP Cron: ✔
Language: en_GB
External object cache: –

### Server Environment ###

Server Info: Apache
PHP Version: 7.4.28
PHP Post Max Size: 64 MB
PHP Time Limit: 90
PHP Max Input Vars: 1000
cURL Version: 7.68.0
OpenSSL/1.1.1f

SUHOSIN Installed: –
MySQL Version: 5.5.5-10.3.34-MariaDB-0ubuntu0.20.04.1
Max Upload Size: 32 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 6.3.1
WC Database Prefix: taglm_
Total Database Size: 248.13MB
Database Data Size: 152.18MB
Database Index Size: 95.95MB
taglm_woocommerce_sessions: Data: 10.02MB + Index: 0.25MB + Engine InnoDB
taglm_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
taglm_woocommerce_order_items: Data: 1.52MB + Index: 0.23MB + Engine InnoDB
taglm_woocommerce_order_itemmeta: Data: 7.52MB + Index: 6.03MB + Engine InnoDB
taglm_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
taglm_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_woocommerce_payment_tokens: Data: 0.05MB + Index: 0.02MB + Engine InnoDB
taglm_woocommerce_payment_tokenmeta: Data: 0.08MB + Index: 0.09MB + Engine InnoDB
taglm_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_actionscheduler_actions: Data: 24.06MB + Index: 28.36MB + Engine InnoDB
taglm_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_actionscheduler_logs: Data: 15.05MB + Index: 13.03MB + Engine InnoDB
taglm_aiowps_debug_log: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_aiowps_events: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_aiowps_failed_logins: Data: 1.52MB + Index: 0.00MB + Engine InnoDB
taglm_aiowps_global_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_aiowps_login_activity: Data: 0.14MB + Index: 0.00MB + Engine InnoDB
taglm_aiowps_login_lockdown: Data: 6.52MB + Index: 0.00MB + Engine InnoDB
taglm_aiowps_permanent_block: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_atum_order_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_atum_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_atum_product_data: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
taglm_automatewoo_abandoned_carts: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
taglm_automatewoo_customers: Data: 0.33MB + Index: 0.50MB + Engine InnoDB
taglm_automatewoo_customer_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_automatewoo_events: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
taglm_automatewoo_guests: Data: 0.13MB + Index: 0.23MB + Engine InnoDB
taglm_automatewoo_guest_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_automatewoo_logs: Data: 0.25MB + Index: 0.34MB + Engine InnoDB
taglm_automatewoo_log_meta: Data: 1.52MB + Index: 0.88MB + Engine InnoDB
taglm_automatewoo_queue: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
taglm_automatewoo_queue_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_commentmeta: Data: 0.08MB + Index: 0.03MB + Engine InnoDB
taglm_comments: Data: 5.52MB + Index: 2.72MB + Engine InnoDB
taglm_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_mailchimp_carts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_mailchimp_jobs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_options: Data: 18.30MB + Index: 2.11MB + Engine InnoDB
taglm_postmeta: Data: 23.55MB + Index: 20.77MB + Engine InnoDB
taglm_posts: Data: 9.47MB + Index: 2.64MB + Engine InnoDB
taglm_sib_model_contact: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_sib_model_country: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_social_users: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_stock_log: Data: 0.48MB + Index: 0.00MB + Engine InnoDB
taglm_termmeta: Data: 0.08MB + Index: 0.11MB + Engine InnoDB
taglm_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_term_relationships: Data: 0.17MB + Index: 0.14MB + Engine InnoDB
taglm_term_taxonomy: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
taglm_tm_taskmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_tm_tasks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_usermeta: Data: 3.50MB + Index: 3.94MB + Engine InnoDB
taglm_users: Data: 0.16MB + Index: 0.16MB + Engine InnoDB
taglm_wcs_payment_retries: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_wc_admin_notes: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_wc_customer_lookup: Data: 0.27MB + Index: 0.25MB + Engine InnoDB
taglm_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_wc_gpf_render_cache: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_wc_order_product_lookup: Data: 1.52MB + Index: 0.50MB + Engine InnoDB
taglm_wc_order_stats: Data: 0.31MB + Index: 0.33MB + Engine InnoDB
taglm_wc_order_tax_lookup: Data: 0.19MB + Index: 0.17MB + Engine InnoDB
taglm_wc_product_attributes_lookup: Data: 0.19MB + Index: 0.22MB + Engine InnoDB
taglm_wc_product_meta_lookup: Data: 1.52MB + Index: 1.17MB + Engine InnoDB
taglm_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_woocommerce_gc_activity: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
taglm_woocommerce_gc_cards: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
taglm_woocommerce_gc_cardsmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_woocommerce_gpf_google_taxonomy: Data: 1.52MB + Index: 0.13MB + Engine InnoDB
taglm_wpmailsmtp_emails_log: Data: 1.52MB + Index: 0.03MB + Engine InnoDB
taglm_wpmailsmtp_tasks_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_wt_iew_action_history: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_wt_iew_cron: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_wt_iew_ftp: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_wt_iew_mapping_template: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_xtraderstockmanager_plugin_category_sync: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
taglm_xtraderstockmanager_plugin_product_sync: Data: 0.11MB + Index: 0.00MB + Engine InnoDB
taglm_xtraderstockmanager_plugin_run_data: Data: 0.48MB + Index: 0.00MB + Engine InnoDB
taglm_yith_wcwl: Data: 0.08MB + Index: 0.02MB + Engine InnoDB
taglm_yith_wcwl_lists: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
taglm_yoast_indexable: Data: 11.52MB + Index: 8.48MB + Engine InnoDB
taglm_yoast_indexable_hierarchy: Data: 1.42MB + Index: 0.67MB + Engine InnoDB
taglm_yoast_migrations: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
taglm_yoast_primary_term: Data: 0.06MB + Index: 0.03MB + Engine InnoDB
taglm_yoast_seo_links: Data: 0.30MB + Index: 0.17MB + Engine InnoDB

### Post Type Counts ###

attachment: 1526
aw_workflow: 5
blocks: 21
custom_css: 1
featured_item: 8
gift_card: 8
gtc_design: 5
gtc_gift_card: 2
gtc_transaction_log: 1
jp_img_sitemap: 2
jp_sitemap: 2
jp_sitemap_master: 2
nav_menu_item: 21
oembed_cache: 1
page: 25
post: 2
product: 253
product_variation: 1346
revision: 230
shop_bogof_rule: 1
shop_coupon: 10
shop_order: 2682
shop_order_refund: 64
shop_subscription: 20
wc_product_tab: 3
wp_global_styles: 1
wpcf7_contact_form: 2
ywtm_tab: 4

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (31) ###

PW Advanced Woo Reporting for "Brand Plugins" ADD-ON: by Proword – 1.1
PW Advanced Woo Reporting: by Proword – 6.0
Akismet Anti-Spam: by Automattic – 4.2.2
All In One WP Security: by Tips and Tricks HQ
Peter Petreski
Ruhul
Ivy – 4.4.10

AutomateWoo: by WooCommerce – 5.5.12
Contact Form 7: by Takayuki Miyoshi – 5.5.6
Custom Adobe Fonts (Typekit): by Brainstorm Force – 1.0.18
Jetpack: by Automattic – 10.7
reCaptcha for WooCommerce: by I Thirteen Web Solution – 2.30
Regenerate Thumbnails: by Alex Mills (Viper007Bond) – 3.1.5
SSH SFTP Updater Support: by TerraFrost
David Anderson + Team Updraft – 0.8.4

Bulk Table Editor for WooCommerce: by Consortia – 2.2.12
WooCommerce Admin: by WooCommerce – 3.2.1
WooCommerce Brands: by WooCommerce – 1.6.28
WooCommerce Buy One Get One Free: by Oscar Gare – 3.1.0
WooCommerce Custom Thank You Pages: by Jeroen Sormani – 1.0.5
WooCommerce Stripe Gateway: by WooCommerce – 6.3.0
WooCommerce Google Analytics Integration: by WooCommerce – 1.5.8
WooCommerce PayPal Payments: by WooCommerce – 1.7.0
WooCommerce Google Product Feed: by Ademti Software Ltd. – 10.6.0
WooCommerce Product Reviews Pro: by SkyVerge – 1.17.4
Sendinblue - WooCommerce Email Marketing: by Sendinblue – 2.0.34
WooCommerce Shipping & Tax: by WooCommerce – 1.25.24
WooCommerce Subscriptions: by WooCommerce – 4.1.0
WooCommerce Xero Integration: by WooCommerce – 1.7.41
WooCommerce: by Automattic – 6.3.1
Yoast SEO: by Team Yoast – 18.4.1
WP-Optimize - Clean, Compress, Cache: by David Anderson
Ruhani Rabin
Team Updraft – 3.2.3

YITH WooCommerce Gift Cards Premium: by YITH – 3.12.1
YITH WooCommerce Tab Manager Premium: by YITH – 1.9.0
YITH WooCommerce Wishlist: by YITH – 3.7.0

### Inactive Plugins (6) ###

Stock Manager for WooCommerce: by StoreApps – 2.8.3
Trustpilot-reviews: by Trustpilot – 2.5.901
Variation Swatches for WooCommerce: by Emran Ahmed – 1.1.19
Variation Swatches for WooCommerce - Pro: by Emran Ahmed – 1.1.18
WooCommerce Gift Cards: by WooCommerce – 1.10.0 (update to version 1.10.2 is available)
XTrader StockManager: by XTrader – 3.43

### Dropin Plugins (1) ###

maintenance.php: maintenance.php

### Settings ###

API Enabled: –
Force SSL: –
Currency: GBP (£)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
gift_card (gift_card)
gift-card (gift-card)
grouped (grouped)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: ✔

### WC Pages ###

Shop base: #63 - /shop/
Basket: #62 - /cart/
Checkout: #61 - /checkout/
My account: #60 - /my-account/
Terms and conditions: #845 - /terms-and-conditions/

### Theme ###

Name: Flatsome Child
Version: 3.0
Author URL: 
Child Theme: ✔
Parent Theme Name: Flatsome
Parent Theme Version: 3.15.4
Parent Theme Author URL: http://www.uxthemes.com/
WooCommerce Support: ✔

### Templates ###

Overrides: flatsome/woocommerce/archive-product.php
flatsome/woocommerce/cart/cart-empty.php
flatsome/woocommerce/cart/cart-shipping.php
flatsome/woocommerce/cart/cart.php
flatsome/woocommerce/checkout/form-checkout.php
flatsome/woocommerce/checkout/form-coupon.php
flatsome/woocommerce/checkout/terms.php
flatsome/woocommerce/checkout/thankyou.php
flatsome/woocommerce/content-product-cat.php
flatsome/woocommerce/content-product.php
flatsome/woocommerce/content-single-product.php
flatsome/woocommerce/content-widget-product.php
flatsome/woocommerce/global/breadcrumb.php
flatsome/woocommerce/global/quantity-input.php
flatsome/woocommerce/global/wrapper-end.php
flatsome/woocommerce/global/wrapper-start.php
flatsome/woocommerce/loop/loop-end.php
flatsome/woocommerce/loop/loop-start.php
flatsome/woocommerce/loop/pagination.php
flatsome/woocommerce/loop/result-count.php
flatsome/woocommerce/loop/sale-flash.php
flatsome/woocommerce/myaccount/form-login.php
flatsome/woocommerce/myaccount/navigation.php
flatsome/woocommerce/notices/error.php
flatsome/woocommerce/notices/notice.php
flatsome/woocommerce/notices/success.php
flatsome/woocommerce/product-searchform.php
flatsome/woocommerce/single-product/photoswipe.php
flatsome/woocommerce/single-product/price.php
flatsome/woocommerce/single-product/product-image.php
flatsome/woocommerce/single-product/product-thumbnails.php
flatsome/woocommerce/single-product/rating.php
flatsome/woocommerce/single-product/related.php
flatsome/woocommerce/single-product/review.php
flatsome/woocommerce/single-product/sale-flash.php
flatsome/woocommerce/single-product/share.php
flatsome/woocommerce/single-product/short-description.php
flatsome/woocommerce/single-product/tabs/tabs.php
flatsome/woocommerce/single-product/title.php
flatsome/woocommerce/single-product/up-sells.php
flatsome/woocommerce/single-product-reviews.php
flatsome/woocommerce/single-product.php
flatsome/woocommerce/content-product_cat.php


### WooCommerce Google Product Feed feeds ###

google: Google merchant centre product feed
  Type: google
  Category filter: -
  Categories: -

### WooCommerce Google Product Feed options ###

Include variations in feed: Enabled
Send "item group ID": -
Expanded schema markup: -
Debug key: f0cbe72f-b800-496e-b8a5-b5a0c292fcc1

### WooCommerce Google Product Feed fields ###

Title: Pre-populates from product product_title.
Product description: Main product description (full preferred) plus variation description
Availability (in stock products): Defaults to "in stock".
Availability (backordered products): Defaults to "in stock".
Availability (out of stock products): Defaults to "out of stock".
Condition: Defaults to "new".
Brand: Pre-populates from product_brand taxonomy.
Manufacturer Part Number (MPN): Pre-populates from product sku.
Product Type: Pre-populates from product_cat taxonomy.
Google Product Category: 
Gender: Defaults to "male".
Age Group: 
Colour: Pre-populates from pa_colour taxonomy.
Size: Pre-populates from pa_size taxonomy.
Size system: Defaults to "UK".
Product detail(s): 

### WooCommerce Google Product Feed DB status ###

Database version: 15
Active database version: 15
wc_gpf_render_cache: Empty
woocommerce_gpf_google_taxonomy (en-GB): 5595 items

### Subscriptions ###

WCS_DEBUG: ✔ No
Subscriptions Mode: ✔ Live
Subscriptions Live URL: https://loadedmenswear.co.uk
Subscription Statuses: wc-active: 13
wc-cancelled: 7

WooCommerce Account Connected: ✔ Yes
Active Product Key: ✔ Yes
Custom Retry Rules: ✔ No
Custom Retry Rule Class: ✔ No
Custom Raw Retry Rule: ✔ No
Custom Retry Rule: ✔ No
Retries Migration Status: ✔ Completed
Report Cache Enabled: ✔ Yes
Cache Update Failures: ✔ 0 failure

### Store Setup ###

Country / State: United Kingdom (UK)

### Subscriptions by Payment Gateway ###

Stripe: wc-active: 13
wc-cancelled: 7


### Payment Gateway Support ###

Stripe: products
refunds
tokenization
add_payment_method
subscriptions
subscription_cancellation
subscription_suspension
subscription_reactivation
subscription_amount_changes
subscription_date_changes
subscription_payment_method_change
subscription_payment_method_change_customer
subscription_payment_method_change_admin
multiple_subscriptions

PayPal: refunds

### Buy One Get One Free ###

Display eligible free gift(s) on: after_cart
Disable coupons: yes
Custom attributes: no

### WooCommerce PayPal Payments ###

Onboarded: ✔
Shop country code: GB
WooCommerce currency supported: ✔
PayPal card processing available in country: ✔
Pay Later messaging available in country: ✔
Webhook status: ✔
Vault enabled: ✔
Logging enabled: –
Reference Transactions: –
Used PayPal Checkout plugin: –

### Action Scheduler ###

Complete: 68,668
Oldest: 2022-02-24 18:01:15 +0000
Newest: 2022-03-27 18:02:33 +0100

Failed: 1
Oldest: 2021-05-08 08:52:52 +0100
Newest: 2021-05-08 08:52:52 +0100

Pending: 17
Oldest: 2022-03-27 18:18:18 +0100
Newest: 2022-04-20 19:03:52 +0100


### Status report information ###

Generated at: 2022-03-27 18:18:41 +01:00
</details>

**Comment by itsmeichigo** [comment]
Thanks, @erricgunawan for confirming the case! I can't reproduce the issue, so it may be challenging to address it.

Also, it would be great if we could create a separate issue for this since the original issue can be fixed by switching the time range. I'm still leaning on the possibility that this is a sync issue. Is there a way to retrieve the device log to see if the actions we took to fix the sync work? That would be helpful I think.

**Comment by igotdes** [comment]
@itsmeichigo Thank you for helping us with this! I've created the separate issue here: [Incorrect Currency Is Displayed](https://github.com/woocommerce/woocommerce-ios/issues/6540).

**Comment by itsmeichigo** [comment]
Thanks! I'm closing this issue again, will follow up on the new one.
